public/js/components/Menu.jsx

```diff
@@ -46,6 +47,9 @@ var Menu = React.createClass({
                       unleash admin
                    </span>
                   </a>
+                  <div style={{position: "absolute", right: "0"}}>
```

Comment:
Could you replace this style with something from the Styleguide? Maybe float the element with .fright.